src/Microsoft.HttpRepl.Fakes/NullPreferences.cs

```diff
@@ -14,7 +14,7 @@ public class NullPreferences : IPreferences
 
         public AllowedColors GetColorValue(string preference, AllowedColors defaultValue = AllowedColors.None)
         {
-            return defaultValue;
+            return AllowedColors.None;
```

Comment:
If we change AllowedColors.None to be 0x0, this could remain as `default`.  It would also make us comply with [CA1008](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1008-enums-should-have-zero-value?view=vs-2019):

>If an enumeration that has the FlagsAttribute applied defines a zero-valued member, its name should be 'None' to indicate that no values have been set in the enumeration. Using a zero-valued member for any other purpose is contrary to the use of the FlagsAttribute in that the AND and OR bitwise operators are useless with the member. ...

Reply:
I made the changes we discussed offline with making `AllowedColors` more closely align with the spec. But note that it was returning `defaultValue` (the parameter) before, not `default` (as it is now). That's what allowed the Warning/Error colors to sneak in.

Changing `GetIntValue` and `GetValue` to return `default` along with `GetColorValue` really breaks things so I left those alone.